src/platform/windows/nvprefs/nvprefs_interface.cpp

```diff
@@ -1,9 +1,9 @@
-#include "nvprefs_common.h"
-
-#include "nvprefs_interface.h"
+// main include for assert
+#include "src/main.h"
```

Comment:
```suggestion
#include "src/main.h"  // main include for assert
```

This is also local (sunshine provided)